aea/cli/utils/decorators.py

```diff
@@ -162,9 +163,17 @@ def wrapper(click_context, *args, **kwargs):
 
         :return: source method output.
         """
-        ctx = cast(Context, click_context.obj)
+        if type(context) is Context:
+            ctx = context
+        elif type(context) is click.core.Context:
+            ctx = cast(Context, context.obj)
+        else:
+            raise AEAException(
+                "clean_after decorator should be used only on methods with Context "
+                "or click.core.Context object as a first argument."
+            )
         try:
-            return func(click_context, *args, **kwargs)
+            return func(context, *args, **kwargs)
```

Comment:
That won't work as before if `type(context) is Context`

Reply:
Yes, the functionality is expanded. Now it's also allowed to use with ctx as first arg (together with `@pass_ctx` decorator).